tests/test_dependabot_test.py

```
@@ -0,0 +1,419 @@
+            getattr(dependabot_test, name)
+            for name in dir(dependabot_test)
+            if callable(getattr(dependabot_test, name))
+            and not name.startswith("_")
+        ]
+
+        for func in functions:
+            try:
+                func(None)
+            except (TypeError, ValueError, AttributeError):
+                pass
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)